test/memory/allocation/align.chpl

```diff
@@ -0,0 +1,17 @@
+use CPtr;
```

Comment:
I'm fine with leaving these tests here if you want, but if we think it's worthwhile to have more exposure I'd be ok with `test/runtime/configMatters/mem` or something. We don't have that many memory layers anymore, but this is a directory that we're likely to run on different architectures/OSs that might have different aligned allocation implementations.